.travis.yml

```diff
@@ -8,18 +8,18 @@ before_install:
 - export PATH=$PATH:${TRAVIS_BUILD_DIR}/target
 install:
 - sudo apt-get install -y gnupg2
-- mvn --settings .travis/settings.xml install -DskipTests=true -Dmaven.javadoc.skip=true -Dgpg.skip=true -B -V -e
-before_deploy:
-- openssl aes-256-cbc -K $encrypted_4d3aca009c62_key -iv $encrypted_4d3aca009c62_iv -in $GPG_DIR/deploy_site.enc -out $GPG_DIR/deploy_site -d
-- openssl aes-256-cbc -pass pass:$ENCRYPTION_PASSWORD -in $GPG_DIR/pubring.gpg.enc -out $GPG_DIR/pubring.gpg -d
-- openssl aes-256-cbc -pass pass:$ENCRYPTION_PASSWORD -in $GPG_DIR/secring.gpg.enc -out $GPG_DIR/secring.gpg -d
+script:
+- mvn install -DskipTests=true -Dmaven.javadoc.skip=true -Dgpg.skip=true -B -V -e
```

Comment:
I don't think `gpg.skip=true` is needed since the profile with gpg shouldn't be activated here.